app/models/user.rb

```diff
@@ -1,13 +1,29 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
As per the project requirements, the user should also be able to confirm his account. Kindly add `confirmable` property to the list also.